### DISCUSSION TITLE
Can't Publish WAPM Package: "[my.wasm] detected as invalid because bulk memory support is not enabled"

### DISCUSSION THREAD

**Discussion by zicklag** [wasmerio/wasmer#3446] [Q&A] [answered]
Hey there!

I was just trying to publish my first WAPM package, but the CLI is telling me that my WASM file is invalid because bulk memory support is not enabled.

My WASM is built from a simple Rust crate, and I don't know if there's a way to tell Rust not to generate bulk memory instructions or not.

Also, I can run the WASM file just fine with `wasmer run` so I'm not sure what the issue is.

Any pointers would be appreciated!

**Comment by theduke** [comment, marked as answer]
I also ran into this .

I assume you are building on Rust nightly, correct?
This seems to be a recent change in Rust nightly default wasm compilation flags.

I'll create an issue.

In the meantime: things should work fine with stable Rust.

**Reply by zicklag** [reply]
Ah yes, that was it! I had forgotten I was using nightly. Using stable works for me for now, which is good enough, thanks for the pointer!
